server/playbooks/product/playbooks_product.go

```diff
@@ -53,12 +54,10 @@ const (
 
 const ServerKey product.ServiceKey = "server"
 
-// These credentials for Rudder need to be populated at build-time,
-// passing the following flags to the go build command:
-// -ldflags "-X main.rudderDataplaneURL=<url> -X main.rudderWriteKey=<write_key>"
-var (
-	rudderDataplaneURL string
-	rudderWriteKey     string
+// These credentials for Rudder need to be replaced at build-time.
+const (
+	rudderDataplaneURL = "placeholder_rudder_dataplane_url"
```

Comment:
I think the only use case I can imagine would be to swap this out for some testing setup. This will be more easily done when we properly switch back to build flags anyway.